lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -418,6 +419,21 @@ def test_render_html_view_invalid_certificate_configuration(self):
         self.assertIn("Invalid Certificate", response.content)
 
     @override_settings(FEATURES=FEATURES_WITH_CERTS_ENABLED)
+    def test_render_500_view_invalid_certificate_configuration(self):
+        html_view_configurations = CertificateHtmlViewConfiguration.objects.all()
```

Comment:
I think we can update CertificateHtmlViewConfiguration in one go like this `CertificateHtmlViewConfiguration.objects.all().update(enabled=False)`